tests/test_history.py

```diff
@@ -1255,7 +1260,7 @@ def test_daily_splits_and_mergers(self):
                 "close"
             )[asset]
 
-            # first value should be halved, second value unadjusted
+            # first value should be doubled, second value unadjusted
```

Comment:
still looks like it's getting halved (2 -> 1), is this comment change intentional?